src/osd/OSD.cc

```diff
@@ -1935,6 +1935,9 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     f->dump_bool("success", success);
     f->dump_int("value", value);
     f->close_section();
+  } else if (command == "bluestore_db_statistics") {
+    if (store->get_type() == "bluestore")
+      store->get_db_statistics(f);
```

Comment:
dump_objecststore_stats ?

and let's just make get_db_statistics a no-op in ObjectStore and overlaod only in bluestore. Then we don't need the type conditional here.